README.md

```diff
@@ -1,2 +1,79 @@
+[![Build Status](https://badge.buildkite.com/9e0e6c88972a3248a0908506d6946624da84e4e18c0870c4d0.svg)](https://buildkite.com/rust-vmm/vm-virtio-ci)
+
 # vm-virtio
-virtio implementation
+
+The vm-virtio crate provides a [virtio](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=virtio)
```

Comment:
nit: can we use Virtio instead of virtio everywhere in the code/readme? It's triggering my OCD.

Reply:
The correct naming would be `VIRTIO` then.

Reply:
Booo!